include/dxc/Support/d3dx12.h

```diff
@@ -72,43 +72,39 @@ struct CD3DX12_VIEWPORT : public D3D12_VIEWPORT
         MinDepth = minDepth;
         MaxDepth = maxDepth;
     }
-    explicit CD3DX12_VIEWPORT(
```

Comment:
Is there a reason not to delete it and use the version from DirectX-Headers or wherever we get the D3D headers?

Reply:
So, I tried deleting this and using the D3D header, which works locally but fails on the ADO builders. I think that's caused by a mismatch between the version of DX on the builder and the version in the DirectX-Headers repo.

I think there's a bigger problem here that should be solved separately. We haven't taken an update to this header (and can't) so I don't see why it should be a problem for our version to diverge from the original so that we can thin our Windows adapter.

Reply:
We don't yet include DirectX-Headers by default, only for non-windows platforms. That is the ultimate intent though.  The benefit of that is we get the updates cleanly and are using a standard interface.

We can take updates to this header and also the submodule. We just haven't had the need yet.